felix/calc/event_sequencer_test.go

```diff
@@ -116,16 +116,14 @@ var _ = Describe("ParsedRulesToActivePolicyUpdate", func() {
 		// all filled in.  If any are still at their zero value, either the test is out of date
 		// or we forgot to add conversion logic for that field.
 		protoUpdate := calc.ParsedRulesToActivePolicyUpdate(model.PolicyKey{Name: "a-policy"}, &fullyLoadedParsedRules)
-		protoPolicy := *protoUpdate.Policy
-		protoPolicyValue := reflect.ValueOf(protoPolicy)
-		numFields := protoPolicyValue.NumField()
-		for i := 0; i < numFields; i++ {
-			field := protoPolicyValue.Field(i)
-			fieldName := reflect.TypeOf(protoPolicy).Field(i).Name
-			Expect(field.Interface()).NotTo(Equal(reflect.Zero(field.Type()).Interface()),
-				fmt.Sprintf("Field %s in the protobuf struct was still at its zero value; "+
-					"missing from conversion or fully-loaded rule?", fieldName))
+		protoPolicy := protoUpdate.GetPolicy()
+		if googleproto.Equal(protoPolicy, &proto.Policy{}) {
+			return
```

Comment:
Should we do an `Expect` call here instead of `return`?

Reply:
this check is because I observed unit test panic caused by empty proto `protoPolicy.Namespace` reference. Since I updated all reference by using `Get`, this check should be safe to remove.